packages/react-dom/src/events/accumulateTwoPhaseListeners.js

```diff
@@ -76,55 +87,99 @@ export default function accumulateTwoPhaseListeners(
+          ) {
+            // useEvent event listeners
+            const targetType = event.type;
+            const listeners = getListenersFromTarget(instance);
 
-          if (listeners !== null) {
-            const listenersArr = Array.from(listeners);
-            for (let i = 0; i < listenersArr.length; i++) {
-              const listener = listenersArr[i];
-              const {
-                callback,
-                event: {capture, type},
-              } = listener;
-              if (type === targetType) {
-                if (capture === true) {
-                  dispatchListeners.unshift(callback);
-                  dispatchInstances.unshift(node);
-                } else {
-                  dispatchListeners.push(callback);
-                  dispatchInstances.push(node);
+            if (listeners !== null) {
+              const listenersArr = Array.from(listeners);
+              for (let i = 0; i < listenersArr.length; i++) {
+                const listener = listenersArr[i];
+                const {
+                  callback,
+                  event: {capture, type},
+                } = listener;
+                if (type === targetType) {
+                  if (capture === true) {
+                    dispatchListeners.unshift(callback);
+                    dispatchInstances.unshift(node);
+                  } else {
+                    dispatchListeners.push(callback);
+                    dispatchInstances.push(node);
+                  }
                 }
               }
             }
           }
-        }
-        // Standard React on* listeners, i.e. onClick prop
-        if (captured !== null) {
-          const captureListener = getListener(node, captured);
-          if (captureListener != null) {
-            // Capture listeners/instances should go at the start, so we
-            // unshift them to the start of the array.
-            dispatchListeners.unshift(captureListener);
-            dispatchInstances.unshift(node);
+          // Standard React on* listeners, i.e. onClick prop
+          if (captured !== null) {
+            const captureListener = getListener(node, captured);
+            if (captureListener != null) {
+              // Capture listeners/instances should go at the start, so we
+              // unshift them to the start of the array.
+              dispatchListeners.unshift(captureListener);
+              dispatchInstances.unshift(node);
+            }
+          }
+          if (bubbled !== null) {
+            const bubbleListener = getListener(node, bubbled);
+            if (bubbleListener != null) {
+              // Bubble listeners/instances should go at the end, so we
+              // push them to the end of the array.
+              dispatchListeners.push(bubbleListener);
+              dispatchInstances.push(node);
+            }
           }
+        } else if (tag === HostText) {
+          lastHostComponent = instance;
```

Comment:
I think in principle it's okay if we want to skip those (would the old event system catch them? I'm not sure). We should just make sure we don't crash.

Reply:
It won't break. Let me remove this block.